citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/AuthorityServiceTest.java

```diff
@@ -593,4 +598,72 @@ public void test_searchAuthorityEntries_NoAuthorityFound()
         Assert.assertEquals(0, result.getFoundAuthorities().size());
     }
 
+    @Test()
+    public void test_getAll_success() {
+        List<IAuthorityEntry> entriesAlbert = new ArrayList<>();
+        entriesAlbert.add(entry2);
+        entriesAlbert.add(entry3);
+        Page<IAuthorityEntry> entries = new PageImpl<>(entriesAlbert);
+        Mockito.when(entryRepository.findByUsernameOrGroupsInOrderByName(username, groupIds, paging)).thenReturn(entries);
+        Page<IAuthorityEntry> searchResult = managerToTest.getAll(user, groupIds, page, pageSize);
+        Assert.assertEquals(2, searchResult.getContent().size());
+        for (IAuthorityEntry entry : searchResult) {
+            Assert.assertTrue(entry.getGroups().contains(groupId));
+            Assert.assertTrue(entry.getName().contains("Albert"));
+            Assert.assertTrue(!entry.getName().contains("Einstein"));
```

Comment:
what are these asserts testing? the point is that all entries returned by the entryRepository are also returned by the manager, not that the names include "Albert" but not "Einstein".